content/md/products-super-power/product-mass-actions.md

```diff
@@ -85,12 +85,36 @@ In order to save you some time, you can also directly categorize products in cat
 
 | Operations                  | Step of configuration |
 |:----------------------------|:----------------------|
-| Add to categories | The products will be positionned into the selected categories, the existing placement is kept |
+| Add to categories | The products will be positioned into the selected categories, the existing placement is kept |
 | Move between categories     | The products will be moved into the selected categories, the existing placement is lost       |
 | Remove from categories | The products will be removed from the selected categories                                     |
 
 Confirm the action with the `Confirm` button located in the middle of the screen.
 
+# Add products to an existing product model
+
+To ease the enrichment of [products with variants](/articles/what-about-products-variants.html), you can gather products in a product model to allow the enrichment of their common properties.
+
+1.  Search for products in the grid with the appropriate filters
+2.  Select the products to add to an existing product model: Tick the products you want to add via the down arrow of the toolbar: `All` for all the products recorded, `All visible` for all visible products on the ongoing page, `none` for none of the products
```

Comment:
is upper case required for "Tick"?

Reply:
it as a copy/paste, I change it for "tick"